compiler/front/options.nim

```diff
@@ -306,6 +306,9 @@ type
     when defined(nimDebugUnreportedErrors):
       unreportedErrors*: OrderedTable[NodeId, PNode]
 
+const 
+  IdeLocCmds* = {ideSug, ideCon, ideDef, ideUse, ideDus, ideHighlight} ## source location awared IDE commands 
```

Comment:
Two things:
* `ideHighlight` is not a command that uses a cursor position 
* the line is too long